frontend/pages/index.tsx

```diff
@@ -23,21 +23,20 @@ export const getServerSideProps = async (
     headers: { cookie: context.req.headers.cookie },
   })
 
-  if (res.ok) {
-    return {
-      redirect: {
-        destination: DASHBOARD_ROUTE,
-        permanent: false,
-      },
-    }
-  } else {
-    return {
-      redirect: {
-        destination: `/api/user/clear-cookies/?next=${DASHBOARD_ROUTE}`,
-        permanent: false,
-      },
-    }
+  if (
+    res.status === 403 &&
+    (await res.json()).detail ===
+      'Authentication credentials were not provided.'
```

Comment:
not too happy about this hard coding but both errors return a 403, need some way to distinguish...

another thought was making a backend route soley to check if the user was logged into our backend, however thought this was overkill...